app/src/main/java/com/communistutopia/spacetrader/viewmodel/LocationViewModel.kt

```diff
@@ -21,6 +21,7 @@ class LocationViewModel: ViewModel() {
         if(player.value!!.spaceship.canTravelTo(player.value!!.system, destinationSystem)) {
             var dist = player.value!!.system.getDistance(destinationSystem)
             updateLocation(destinationSystem, destinationPlanet)
+            player.value!!.location.rollForRandomEvent()
```

Comment:
Just a note, we should probably have a toast or label somewhere in the MarketView displaying if this event triggers. @DrakeWitt and I could probably look into that.